### ISSUE TITLE
Data explorer graph is really small on IE11

### ISSUE THREAD

**Issue by toolness** [18F/calc#1064]
I'm not sure if something is wonky with my IE11 settings, but I just noticed that the graph on the data explorer is *really* small in IE11:

> ![calc_graph_ie11](https://cloud.githubusercontent.com/assets/124687/20635834/adf16d2c-b330-11e6-880b-905689c07316.png)

In contrast, this is what it looks like in Chrome:

> ![calc_graph_chrome](https://cloud.githubusercontent.com/assets/124687/20635840/b547227e-b330-11e6-91d6-1ba5b904449d.png)

The graph is still this small on calc.gsa.gov, which is currently running the ancient version of CALC that we didn't make many changes to. So either something is messed up with my IE settings, or it's been a bug in CALC since before we started working on it. If it's the latter, it might also explain why some of our user testers have their browser zoom set so high.

**Comment by hbillings** [comment]
@toolness Is this still an issue?

**Comment by toolness** [comment]
Yep, I was just using CALC on IE11 a few days ago and it is still happening, at least on the two Windows machines I have access to.

Furthermore, it looks like a plurality of CALC's users are on IE11, so this might warrant some deeper investigation:

> ![browsers](https://user-images.githubusercontent.com/124687/41227040-7b3c30ac-6d41-11e8-8426-8074374b1fd8.png)

(Note that 97% of those IE users are on IE11, the rest are on IE7 apparently o_O)

**Comment by ericronne** [comment]
➕ to prioritizing! (and to establishing a QA protocol)

Cheaper than distributing macbooks to all users.
